.eslintrc.json

```diff
@@ -23,6 +23,7 @@
     "security/detect-non-literal-fs-filename": "off",
     "security/detect-object-injection": "off",
 
+    "no-console": "off",
```

Comment:
I'm guessing this was put in place to troubleshoot tests, and just got missed with the last push.  This option needs to be removed so we keep the default for "no-console" as "error".